.gitlab/tar.sh

```diff
@@ -10,7 +10,7 @@ ls -la out/
 TARBALL_PREFIX="haskell-language-server"
 case "${TARBALL_EXT}" in
     zip)
-		HLS_VERSION="$("$CI_PROJECT_DIR/out/haskell-language-server-wrapper" --numeric-version)"
+		HLS_VERSION="$("$CI_PROJECT_DIR/out/haskell-language-server-8.10.7" --numeric-version)"
```

Comment:
Why this change? This will break CI once 8.10.7 is deprecated, no?

Reply:
If hls-wrapper is compiled with 9.2.2 then the job fails because it can't find libgcc_s_seh-1.dll

Reply:
Yeah, this should have been fixed upstream weeks ago.